src/main/java/net/liquidpineapple/pang/objects/ScoreSystem.java

```diff
@@ -35,11 +35,10 @@ public ScoreSystem() {
     }
 
 
-    public void AddScore(int scoreIn) {
+    public static void AddScore(int scoreIn) {
         if(score != 999999999){
```

Comment:
This will only work if your score ends up as exactly 999999999, which is unlikely as the score will not be increased by small enough values.